apps/explorer/components/Contract/index.tsx

```diff
@@ -76,22 +89,32 @@ export function Contract({
         label: 'merkle root',
         value: contract.fileMerkleRoot,
       },
-      !('v2FileContract' in contract) && {
-        label: 'unlock hash',
-        value: contract.unlockHash,
-      },
+      contract.version === 'v1'
+        ? {
+            label: 'unlock hash',
+            value: contract.unlockHash,
+          }
+        : {
+            label: 'resolution type',
+            value:
+              contractResolutionLabels[
+                contract.resolutionType as keyof typeof contractResolutionLabels
+              ] ?? '-',
+            copyable: false,
+          },
       {
-        label: 'proof confirmed',
+        label: 'Resolution Txn',
```

Comment:
Looks like we have a mix of lower case, capital case, sentence case on the labels